src/ResourceManager/ServiceFabric/Commands.ServiceFabric.Test/ScenarioTests/TestServiceFabric.cs

```diff
@@ -34,14 +34,14 @@ public TestServiceFabric(ITestOutputHelper output)
             ServiceFabricCmdletBase.TestThumbprint = "2F51AC39C590551FC7391A7A0A187A67BF8256CA";
         }
 
-        [Fact]
+        [Fact(Skip = "Need service team to re-record test after changes to the Dependencies.")]
```

Comment:
@praries880 we should make sure that @juhacket is aware of these tests being skipped and can manually validate the changes being made in this PR with sample bits

Reply:
I have approved this change. I need to re-record all tests.

Reply:
After reviewing the extent of these changes I think we should try to re-enable the scenario tests. @praries880 Please work directly with me to record the tests.